packages/resourceful-redux/src/utils/reducer-generator.js

```diff
@@ -8,19 +8,19 @@ import warning from './warning';
 // Basically, two things can change:
 //
 // 1. Request status for resource IDs in `meta`, if IDs are passed in
-// 2. Request status for a labeled request
+// 2. Request status for a named request
 //
-// A label's IDs don't change, and neither does the resource. Consequently, this
-// helper completely defines all of the ways in which the non-success reducers
+// A named requests's IDs don't change, and neither does the resource. Consequently,
```

Comment:
`A named request's IDs don't change`

Reply:
might be missing this but the comment says exactly what your comment says?

Reply:
requests’s => request’s